services/static-webserver/client/source/class/osparc/desktop/credits/BuyCreditsInput.js

```diff
@@ -10,71 +10,112 @@ qx.Class.define("osparc.desktop.credits.BuyCreditsInput", {
 
   construct(pricePerCredit, currencySymbol = "$") {
     this.base(arguments);
-    this.__currencySymbol = currencySymbol
-    this.__pricePerCredit = pricePerCredit
+
+    if (isNaN(pricePerCredit)) {
+      return;
+    }
+
+    this.__pricePerCredit = pricePerCredit;
+    this.__currencySymbol = currencySymbol;
     this._setLayout(new qx.ui.layout.HBox(25).set({
       alignX: "center"
-    }))
-    this._render()
+    }));
+    this._render();
   },
 
   events: {
     "input": "qx.event.type.Data"
   },
 
+  statics: {
+    MINIMUM_TOTAL: 10
```

Comment:
you do not use this anymore,right? Otherwise you could use it in the default above? i.e. instead of the `10` in 
`const minimum = "minPaymentAmountUsd" in data ? data["minPaymentAmountUsd"] : 10;`